src/components/ui/data-table-column-header.tsx

```diff
@@ -0,0 +1,90 @@
+"use client"
+
+import {
+    ArrowDownIcon,
+    ArrowUpIcon,
+    CaretSortIcon,
+    ResetIcon,
+} from "@radix-ui/react-icons"
+import {Column} from "@tanstack/react-table"
+
+import {Button} from "@/components/ui/button"
+import {
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuItem,
+    DropdownMenuSeparator,
+    DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu"
+
+import {cn} from "@/lib/utils"
+import React from "react"
+
+interface DataTableColumnHeaderProps<TData, TValue>
+    extends React.HTMLAttributes<HTMLDivElement> {
+    column: Column<TData, TValue>
+    title: string
+    hasDropDownMenu?: boolean
+}
+
+export function DataTableColumnHeader<TData, TValue>({column, title, className, hasDropDownMenu = true}: DataTableColumnHeaderProps<TData, TValue>) {
```

Comment:
awesome, thanks for modifying component so that columns have option for no dropdown